packages/terra-notification-dialog/CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ Changelog
 
 Unreleased
 ----------
+### Breaking Changes
+See Updated Guide for more info: 
+
+* Updated `variant` prop to be required
+* Removed `variant="success"` / `variant={NotificationDialogVariants.SUCCESS}` from variant options
+* Relaxed `isOpen` prop to be optional
+* Added `custom` prop
+* Removed `header` prop
+* Removed `customIcon` prop
+* Removed deprecated `primaryAction` prop
+* Removed deprecated `secondaryAction` prop
```

Comment:
Looks like the `message` prop was also removed

Reply:
added: https://github.com/cerner/terra-framework/pull/1105/commits/1c8f6e93521b125790227ff706bd1fda8d9baa68